### ISSUE TITLE
BUG: python library not loaded, no LC_RPATH's found

### ISSUE THREAD

**Issue by FebianFebian1** [oasysai/oasysdb#49]
### Short description

The bug happens when running `cargo test` in the terminal. Reason: no LC_RPATH's found. 

### Steps to reproduce

The bug happens after following this steps below from nothing: 
(Running on VS Code, MacOS 14.2 M2 Chips, Python Version installed 3.11.8)

1. Installing rust following the steps listed in this website: https://www.rust-lang.org/tools/install
2. Extension Rust Analyzer is installed inside the VSCode.
3. The Oasysdb main branch repo is forked and clone into local machine.
4. Change directory to the cloned folder of "oasysdb"
5. Then run `cargo test` from the terminal. 

### Expected Behavior

Expected: The compilation works successfully and giving no errors
Error encountered: Library not loaded: @rpath/Python3.framework/Versions/3.9/Python3, error: test failed, to rerun pass `--lib`. 

### Severity

This might blocks the new users or contributors to use or contribute. 

### Additional Context

Screenshot of the full error is provided here:
<img width="1085" alt="OasysDB Error" src="https://github.com/oasysai/oasysdb/assets/51969569/1aa34ec9-f0d3-4752-b248-558c2509194a">

**Comment by edwinkys** [comment]
So this problem seems to occur due to Maturin and PyO3 setup to get started with the project. With this, I have actually update the [contributing.md](https://github.com/oasysai/oasysdb/blob/main/docs/contributing.md) documentation to include the set up that you can follow to get started to work on the project. Check out the `Getting started` section.

### TL;DR

1. Install Maturin.
2. Set up and activate Python virtual environment.
3. Re-run the tests.
